canonical-web-and-design/jaas-dashboard
# Add model data counts to top of the controller view

## Done
- Add model data counts to top of the controller view
- Capitalise the True or False for public column to match the design

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to controllers
- See the counts of machines, applications and units appear at the top 

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1073
